js/src/common/index.js

```diff
@@ -1,6 +1,8 @@
-import 'expose-loader?$!expose-loader?jQuery!jquery';
-import 'expose-loader?m!mithril';
-import 'expose-loader?dayjs!dayjs';
+// Expose jQuery, mithril and dayjs to the window browser object
+import 'expose-loader?exposes[]=$&exposes[]=jQuery!jquery';
+import 'expose-loader?exposes=m!mithril';
```

Comment:
This is a nicer API, I like this change

Reply:
Me too! v2 has an even nicer API too!

```js
import 'expose-loader?exposes=$,jQuery!jquery';
```

Reply:
> Me too! v2 has an even nicer API too, but that's Webpack 5 only :(

We'll need to update to Webpack 5 for the automated export stuff. Supporting it isn't difficult at all either, see https://github.com/flarum/core/compare/as/support-webpack-5